packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -20,17 +20,14 @@ import { PBRMaterial } from "core/Materials/PBR/pbrMaterial";
 import { CubeTexture } from "core/Materials/Textures/cubeTexture";
 import { Texture } from "core/Materials/Textures/texture";
 import { Color4 } from "core/Maths/math.color";
-import { Scalar } from "core/Maths/math.scalar";
+import { Clamp } from "core/Maths/math.scalar.functions";

```

Comment:
A quick FYI here - this will fail if you ever build the package in UMD, as these functions are not exported directly from the .functions file. Just as a general note as I know you don't build a UMD version of the viewer.

Reply:
Ah wasn't aware, good to know. Curious though, why don't we export this from the index.ts?

Reply:
to avoid over-populating the global namespace with all of the single functions